pipeline/transform.py

```diff
@@ -177,6 +177,14 @@ def is_valid_url(url: str) -> str:
     return url
 
 
+def standardise_author_url(url):
```

Comment:
i think there is a missing type hint here for the return value and for the argument.
```suggestion
def standardise_author_url(url: str) -> str:
```